app/webpacker/components/Tickets/index.jsx

```diff
@@ -79,9 +21,16 @@ export default function Wrapper({ id }) {
 }
 
 function Tickets({ id }) {
+  const queryClient = useQueryClient();
   const {
-    data: ticketDetails, sync, loading, error,
-  } = useLoadedData(actionUrls.tickets.show(id));
+    data: ticketDetails,
+    isLoading: isLoadingTicketDetails,
```

Comment:
Here you should probably use `isPending`. The flag `isLoading` only covers the first, initial queryFn upon page load.